composer.json

```diff
@@ -12,6 +12,7 @@
         }
     ],
     "autoload": {
+        "files": ["src/functions_include.php"],
```

Comment:
It's a bit of an odd thing to have happen, but I'm not sure what would happen if a project had 2 dependencies that both depended on this lib. [Composer uses `include`](https://github.com/composer/composer/blob/master/src/Composer/Autoload/ClassLoader.php#L412) rather than `include_once`.

TBH I just did the same exact thing Guzzle does, and I don't think it really hurts to check.